posydon/binary_evol/SN/step_SN.py

```diff
@@ -1726,7 +1756,82 @@ def generate_kick(self, star, sigma):
             Vkick = 0.0
 
         return Vkick
+    def rotate(axis, angle):
+
+        """Generate rotation matrix to rotate a vector about an arbitrary axis 
+            by a given angle
```

Comment:
The rotate function is a general rotation. Hence, I'm wondering about two things:
1. Shouldn't there be a similar function already existing? I haven't seen one in numpy, but it looks like scipy has. Thus, may consider to not define this ourselves.
2. If we define it ourselves, is it specific to the SN_step or should we better put it in the common_functions.py in utils?

Reply:
If Chase has already built the function and it works, I don't mind keeping what we have put together already. But I agree it should go in common_functions.py.

Reply:
As suggested, I've moved rotate() this to common_functions